src/modules/analysis/AnalysisAddComponent.js

```diff
@@ -142,6 +143,42 @@ const AnalysisAdd = (
     <PageHeader style={{ paddingLeft: 20 }}>
       {I18n.t('musit.analysis.registeringAnalysis')}
     </PageHeader>
+    <Form horizontal style={{ paddingLeft: 10 }}>
+      <FormGroup>
+        <Label label="Type analyse" md={1} />
+        <Col md={2}>
+          <select
```

Comment:
We have used it several places in Analysis and Samples so we can have separate PR to make them similar.